RecoVertex/AdaptiveVertexFinder/plugins/TemplatedVertexArbitrator.h

```diff
@@ -54,6 +54,25 @@ class TemplatedVertexArbitrator : public edm::stream::EDProducer<> {
 	typedef std::vector<VTX> Product;
 	TemplatedVertexArbitrator(const edm::ParameterSet &params); 
 
+	static void fillDescriptions(edm::ConfigurationDescriptions & cdesc) {
+	  edm::ParameterSetDescription pdesc;
+	  pdesc.add<edm::InputTag>("beamSpot",edm::InputTag("offlineBeamSpot"));
+	  pdesc.add<edm::InputTag>("primaryVertices",edm::InputTag("offlinePrimaryVertices"));
+	  pdesc.add<edm::InputTag>("tracks",edm::InputTag("particleFlow"));
+	  pdesc.add<edm::InputTag>("secondaryVertices",edm::InputTag("candidateVertexMerger"));
+	  pdesc.add<double>("dLenFraction",0.3333);
+	  pdesc.add<double>("dRCut",0.4);
+	  pdesc.add<double>("distCut",0.04);
+	  pdesc.add<double>("sigCut",5.0);
+	  pdesc.add<double>("fitterSigmacut",3.0);
+	  pdesc.add<double>("fitterTini",256);
+	  pdesc.add<double>("fitterRatio",0.25);
+	  pdesc.add<int>("trackMinLayers",4);
+	  pdesc.add<double>("trackMinPt",0.4);
+	  pdesc.add<int>("trackMinPixels",1);
+	  pdesc.add<double>("maxTimeSignificance",3.5);
+	  cdesc.addDefault(pdesc);
```

Comment:
please complete migration to fillDescriptions: make the system generate a default config here (e.g. templatedVertexArbitratorDefault) and then use it.